### PR TITLE
Mdkessler patch 14

### PR DESCRIPTION
**Pull request by mdkessler**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**

### CODE REVIEW

**Review by Benjamin-Lee — APPROVED** [review]

**Comment by Benjamin-Lee** [comment]
Thanks for reviewing all the tips @mdkessler! This was a ton of work on your part and I'm sure I speak for everyone when I say I'm really grateful for the revisions

File: content/13.conclusion.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Conclusion {#conclusion}
 
-Deep learning techniques have the potential for wide use in biology, meeting or exceeding the performance of both humans and the current state-of-the art algorithms in a variety of tasks.
-Beyond simply achieving good predictive performance, deep learning has the potential to generate novel biological insights that could assist the progress of fundamental research.
+Deep learning techniques have the potential for wide use in biology, and the capacity to meet or exceed the performance of both humans and the current state-of-the art algorithms across a wide range of tasks.
```
Comment (cgreene) [inline content/13.conclusion.md:3]:
```suggestion
Deep learning techniques have the potential for wide use in biology and the capacity to meet or exceed the performance of both humans and the current state-of-the art algorithms across a wide range of tasks.
```

File: content/13.conclusion.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+Beyond simply achieving good predictive performance, deep learning also has the potential to inform novel biological insights, and to therefore fundamentally drive high value research progress.
```
Comment (cgreene) [inline content/13.conclusion.md:4]:
```suggestion
Beyond simply achieving good predictive performance, deep learning has the potential to inform novel biological insights to fundamentally drive high value research.
```

**Review by cgreene — APPROVED** [review]
A few small changes - looks good after they get merged

**Pull request merged**